tests/test_deserialization.py

```diff
@@ -565,10 +581,27 @@ def test_timestamp_field_deserialization(self, fmt, value, expected):
         ["", "!@#", 0, -1, dt.datetime(2013, 11, 10, 1, 23, 45)],
     )
     def test_invalid_timestamp_field_deserialization(self, fmt, in_value):
-        field = fields.DateTime(format="timestamp")
+        field = fields.DateTime(format=fmt)
         with pytest.raises(ValidationError, match="Not a valid datetime."):
             field.deserialize(in_value)
 
+    @pytest.mark.parametrize("fmt", ["timestamp", "timestamp_ms"])
+    def test_oversized_timestamp_field_deserialization(self, fmt):
+        field = fields.DateTime(format=fmt)
+        with pytest.raises(ValidationError, match="Not a valid datetime."):
```

Comment:
I just realized this error message is not that for timestamps. But that's another story.

Reply:
Do you mean it should return `"Not a valid timestamp."` when failing to deserialize a timestamp ?